packages/salesforcedx-vscode-core/test/commands/commands.test.ts

```diff
@@ -53,7 +53,7 @@ describe('Command Utilities', () => {
         new class implements ParametersGatherer<{}> {
           public async gather(): Promise<
             CancelResponse | ContinueResponse<{}>
-          > {
+            > {
```

Comment:
Did something change with the prettier algorithm here? It looks to be mostly whitespace issues.

Reply:
I accidentally prettier disabled the last time I made changes to this file, so I enabled it and fixed the files with bad formatting.